packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -131,7 +131,7 @@ export default class Node extends PureComponent {
                     isHidden={$get('properties._hidden', node)}
                     isHiddenInIndex={$get('properties._hiddenInIndex', node)}
                     hasError={this.hasError()}
-                    label={$get('label', node)}
+                    label={$get('label', node).replace(/<\/?[^>]+(>|$)/g, '')}
```

Comment:
Would be nice to put this strip tags logic to some utility class, in case we would want to reuse it again. Or even use some npm lib for this...